gradle.properties

```diff
@@ -89,14 +89,14 @@ cassandra-all.version=4.0.1
 cassandra-java-driver-core.version=4.13.0
 cassandra-unit.version=4.3.1.0
 chinook-data-hsqldb.version=0.2
-commons-codec.version=1.16.0
-commons-dbcp2.version=2.11.0
-commons-io.version=2.15.0
-commons-lang3.version=3.13.0
+commons-codec.version=1.13
```

Comment:
These changes are unrelated to this PR, I don't think they should be here.
Are you missing a rebase?

Reply:
Because he want test `Revert "Bump 6 dependencies"`.

Now, we know druid test failed is not about it.

So you can drop  the revert commit.